modules/swagger-codegen/src/main/resources/JavaSpringMVC/pom.mustache

```diff
@@ -121,10 +121,10 @@
     <swagger-core-version>1.5.8</swagger-core-version>
     <jetty-version>9.2.9.v20150224</jetty-version>
     <jersey-version>1.13</jersey-version>
```

Comment:
I do not find what this dependency property is used . Never used ? Delete ?
http://mvnrepository.com/artifact/io.swagger/swagger-jersey-jaxrs/1.5.8